roxie/ccd/ccdmain.cpp

```diff
@@ -773,7 +772,7 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                 {
                     // Makes debugging easier...
                     IPropertyTree *service = topology->addPropTree("RoxieFarmProcess");
-                    service->setPropInt("@port", 9876);
+                    service->setPropInt("@port", topology->getPropInt("@port", 9876));
```

Comment:
Should this be a separate bug?  I don't mind if it stays here, but nice to know what it is fixing.

Reply:
It just makes testing easier. This codepath is only for developer test usage.

Reply:
(means I can have two roxie's running in local server mode at the same time, to compare behaviour between versions, for example)